src/test/java/jenkins/plugins/openstack/pipeline/OpenstackMachineStepTest.java

```diff
@@ -97,11 +98,14 @@ public void bootInvalidTemplate() throws Exception {
     @Test
     public void checkSerializableSimplifiedServer() throws Exception {
 
+        final String echoIpStepWindows = "bat \"echo Instance IP: ${srv.address}\"";
+        final String echoIpStepUnix = "sh \"echo Instance IP: ${srv.address}\"";
+        final String echoIpStep = Functions.isWindows() ? echoIpStepWindows : echoIpStepUnix;
         WorkflowJob bootSerializableCheck = j.jenkins.createProject(WorkflowJob.class, "bootSerializableCheck");
         bootSerializableCheck.setDefinition(new CpsFlowDefinition(
                 " def srv = openstackMachine cloud: 'openstack', template: 'template0'\n" +
                         "node ('master') { \n" +
-                        "  sh \"echo Instance IP: ${srv.address}\" \n" +
```

Comment:
I guess there is no reason to wrap this into a `sh`/`bat`. [`echo`](https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-echo-code-print-message) step would do I guess.